ruby-3.1.2-node-16/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ruby:3.1.2
+
+# Node
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - 
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list 
+RUN apt-get update 
+RUN apt-get install -y nodejs yarn p7zip-full
```

Comment:
I would move p7zip-full into the project and only kept here a common dependencies